portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,10 +41,9 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int maxComments;
```

Comment:
I'd structure this differently. Notice you can do:
int maxComments = 0;
if (maxCommentsParam != null) { maxComments = Integer.parseInt(..);}
// use maxComments.

This is the same logic, yet saves you the else branch.

Reply:
much better!